doc/decisions/DR014-Management-policy-responses.md

```diff
@@ -0,0 +1,256 @@
+# DR014 Management Policy responses
+
+- **Status:** Proposed
+- **Driver:** @foundrytom
+- **Approver:** @feltech @ecmorris @antirotor @mattdaw
+- **Outcome:** A Manager should additionally imbue a `managementPolicy`
+  response with the traits that it is capable of resolving/persisting
+  for each trait set.
+
+## Background
+
+The ability for a host to query a manager's capabilities and intent is a
+critical part of the OpenAssetIO abstraction.
+
+OpenAssetIO classifies entities (assets) using a Trait Set, which lists
+one or more traits that describe it's nature. For example, and image
+file may have the trait set `{'file', 'image', 'raster',
+'colorManaged'}`.
+
+The `managementPolicy` method is a high-level (i.e. not entity specific)
+query that a host can use to determine a manager's behaviour in relation
+to a specific trait set.
+
+The answer to this query is commonly used by a Host to conditionally
+enable user-facing functionality based on the behaviour of the specific
+manager that is in use.
+
+This mechanism is flexible, and so exactly how a manager should
+implement its response is potentially ambiguous. This document outlines
+the rationale behind the defined canonical behaviour.
+
+## Relevant data
+
+Documentation pertaining to the OpenAssetIO data model:
+
+- [Entities, Traits and Specifications overview](../doxygen/src/EntitiesTraitsSpecifications.dox)
```

Comment:
I left this as a file link vs a hyperlink to the GitHub pages version, so it was repo local and should still work in the future, if an old revision is checked out.

Reply:
Interestingly, Doxygen's markdown support includes [`@ref`s](https://doxygen.nl/manual/markdown.html#md_links), so we could one day migrate the standalone Doxygen pages so they can render in either Doxygen or markdown viewers (though obviously `@ref`s won't work in pure markdown).